tests/foreman/cli/test_partitiontable.py

```diff
@@ -1,35 +1,46 @@
 # -*- encoding: utf-8 -*-
 """Test class for Partition table CLI"""
-from fauxfactory import gen_string, gen_alphanumeric
+from random import randint
+from robottelo.datafactory import generate_strings_list
+from robottelo.cli.base import CLIReturnCodeError
 from robottelo.cli.factory import make_os, make_partition_table
-from robottelo.cli.operatingsys import OperatingSys
 from robottelo.cli.partitiontable import PartitionTable
-from robottelo.decorators import run_only_on, tier1, tier2
+from robottelo.decorators import run_only_on, skip_if_bug_open, tier1, tier2
 from robottelo.test import CLITestCase
 
 
 class PartitionTableUpdateCreateTestCase(CLITestCase):
-    """Test case for CLI tests."""
+    """Partition Table CLI tests."""
 
-    def setUp(self):  # noqa
-        """Set up file"""
-        super(PartitionTableUpdateCreateTestCase, self).setUp()
-        self.content = "Fake ptable"
-        self.name = gen_alphanumeric(6)
-        self.args = {'name': self.name,
-                     'content': self.content}
+    @skip_if_bug_open('bugzilla', 1229384)
+    @tier1
+    def test_verify_bugzilla_1229384(self):
```

Comment:
`test_positive_crate_one_character_name`? The bugzilla number is already present on the skip decorator and the `@bz` testimony tag. When it is closed, we will be able to remove all references to the BZ and leaving a test with a meaning name is better.

This is not a blocker, but I think we should avoid requiring to get more information when seeing the results. For example, on jenkins seeing a that a creation with one char in name have failed is easier than requiring to check the code for the description or even checking the related BZ for more information.